enrolments/tests/test_validators.py

```diff
@@ -76,6 +77,43 @@ def test_validate_enrolments__invalid(self, mock_validate):
         self.assertRaises(ValidationError, validate_enrolment, enrolment)
         mock_validate.assert_called_once_with(self.class_in_session, self.session)
 
+    def test_validate_students_in_enrolment(self):
+        parent = Student.objects.create(
+            first_name="Daddy",
+            last_name="McDonald",
+            role=Student.PARENT,
+            family=self.family,
+        )
+        child1 = Student.objects.create(
+            first_name="Lil Tom",
+            last_name="McDonald",
+            role=Student.CHILD,
+            family=self.family,
+        )
+        child2 = Student.objects.create(
+            first_name="Yohan Wilkshire",
+            last_name="McDonald",
+            role=Student.CHILD,
+            family=self.family,
+        )
+        student_ids = list(map((lambda student: student.id), [parent, child1, child2]))
```

Comment:
would this be lil more concise? `[student.id for student in [parent, child1, child2]`

Reply:
yeah that's easier to read as well